src/betterproto/__init__.py

```diff
@@ -1124,6 +1132,15 @@ def SerializeToString(self: T) -> bytes:
         """
         return bytes(self)
 
+    def __getstate__(self) -> bytes:
+        return bytes(self)
+
+    def __setstate__(self: T, pickled_bytes: bytes) -> T:
+        return self.parse(pickled_bytes)
+
+    def __reduce__(self) -> Union[str, Tuple[Any, ...]]:
+        return (self.__class__.FromString, (bytes(self),))
```

Comment:
```suggestion
    def __reduce__(self) -> Tuple[Any, ...]:
        return (self.__class__.parse, (bytes(self),))
```

Reply:
I don't think I can do this, because parse is a bound method
```
Could not bind method "parse" because "Type[Message]" is not assignable to parameter "self"
```
If I use parse, the pickling tests fail with 
```
TypeError: Message.parse() missing 1 required positional argument: 'data'
```
Not really sure what to provide for the self argument

Reply:
Got it, thanks